packages/gestalt/src/Touchable.css

```diff
@@ -1,5 +1,8 @@
 .touchable {
   composes: accessibilityOutline from "./Focus.css";
+  display: block;
```

Comment:
@jackhsu978 do we need `block` here? Since it is a `div`, won't the default `display` be `block`?

Reply:
I don't think we need to. will remove.